### ISSUE TITLE
`// zlint-disable line-length` not working

### ISSUE THREAD

**Issue by gjuchault** [DonIsaac/zlint#327]
**Version**
v0.7.9

**Describe the bug**
It appears that I get warnings about line-length while I did mark the previous line has `// zlint-disable line-length -- can't fit`

**To Reproduce**
Example code:
```zig
// zlint-disable line-length -- test naming
test "some super long test definition some super long test definition some super long test definition some super long test definition" {
  try std.testing.expect(true);
}
```

Example code 2:
```zig
return SomeStruct{
  // zlint-disable line-length -- can't get this to fit
  .some_field = source.field_with_a_super_long_property_name orelse source2.field_with_a_super_long_property_name,
}
```

Configuration:
```
{
    "rules": {
        "case-convention": "warn",
        "line-length": [
            "warn",
            {
                "max-length": 100
            }
        ],
        "no-return-try": "warn",
        "returned-stack-reference": "error",
        "useless-error-return": "warn"
    }
}
```

**Expected behavior**
No warn

**Screenshots**

<img width="1047" height="265" alt="Image" src="https://github.com/user-attachments/assets/65f21bd4-3c4f-40f2-b033-286817592286" />
